components/agendor/actions/create-organization/create-organization.mjs

```diff
@@ -0,0 +1,141 @@
+
+export default {
+  name: "Create Organization",
+  description: "Create Organization [See the documentation](https://api.agendor.com.br/docs/#operation/Create%20organization).",
+  key: "agendor-create-organization",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    name: {
+      type: "string",
+      label: "Name",
+      description: "Display name",
+    },
+    legalName: {
+      type: "string",
+      label: "Legal Name",
+      description: "Legal name",
+      optional: true,
+    },
+    cnpj: {
+      type: "string",
+      label: "Legal Document Number (CNPJ)",
+      description: "Legal document number (CNPJ)",
+      optional: true,
+    },
+    description: {
+      propDefinition: [
+        app,
+        "description",
+      ],
+    },
+    logo: {
+      type: "string",
+      label: "Logo",
+      description: "URL of the logo",
+      optional: true,
+    },
+    website: {
+      type: "string",
+      label: "Website",
+      description: "URL of the website",
+      optional: true,
+    },
+    ranking: {
+      propDefinition: [
+        app,
+        "ranking",
+      ],
+    },
+    onwerUser: {
+      propDefinition: [
+        app,
+        "userId",
+      ],
+    },
+    contact: {
+      propDefinition: [
+        app,
+        "contact",
+      ],
+    },
+    address: {
+      propDefinition: [
+        app,
+        "address",
+      ],
+    },
+    leadOrigin: {
+      propDefinition: [
+        app,
+        "leadOrigin",
+      ],
+    },
+    category: {
+      propDefinition: [
+        app,
+        "category",
+      ],
+    },
+    sector: {
+      propDefinition: [
+        app,
+        "sector",
+      ],
+    },
+    products: {
+      propDefinition: [
+        app,
+        "product",
+      ],
+      type: "string[]",
+      label: "Products",
+    },
+    allowedUsers: {
+      propDefinition: [
+        app,
+        "userId",
+      ],
+      type: "string[]",
+      label: "Allowed Users",
+      description: "Array of IDs of users that should be able to see this organization.",
+    },
+    allowToAllUsers: {
+      propDefinition: [
+        app,
+        "allowToAllUsers",
+      ],
+    },
+    customFields: {
+      propDefinition: [
+        app,
+        "customFields",
+      ],
+    },
+  },
+  async run({ $ }) {
+    const organization = await this.app.createOrganization({
+      name: this.name,
+      legalName: this.legalName,
+      cnpj: this.cnpj,
+      description: this.description,
+      logo: this.logo,
+      website: this.website,
+      ranking: this.ranking,
+      ownerUser: this.ownerUser,
+      contact: this.contact,
+      address: this.address,
+      allowedUsers: this.allowedUsers,
+      allowToAllUsers: this.allowToAllUsers,
+      customFields: this.customFields,
+      leadOrigin: this.leadOrigin,
+      category: this.category,
+      sector: this.sector,
+      products: this.products,
+    });
```

Comment:
```suggestion
  const = {
      app,
      ...data
    } = this;
    
    const organization = await app.createOrganization(data);
```

I think it's better to use this way, what do you think?

Reply:
Yeah, indeed, tks!